shared/featureconfig/flags.go

```diff
@@ -418,11 +414,6 @@ var (
 		Usage:  deprecatedUsage,
 		Hidden: true,
 	}
-	deprecatedEnableInitSyncWeightedRoundRobin = &cli.BoolFlag{
```

Comment:
Why delete this deprecated flag? It will break people's workflows if someone if using it

Reply:
I though that's the way we do it:
![image](https://user-images.githubusercontent.com/188194/85745984-caea4f80-b70e-11ea-8381-f9ef96e34be1.png)

The 'enableInitSyncWeightedRoundRobin' was **opt in** flag. It was deprecated, once we introduced opt-out. Deprecated means users get messages, like above: don't use we'll delete it. Now when **opt out** flag is deprecated, opt-in flag is obsolete and to be deleted. Opt out flag will take some time - but eventually will be deleted too.

Am I missing sth?

Reply:
Ah sorry about that, misunderstood! Thanks @farazdagi